load_testing/locust_helpers.py

```diff
@@ -1,18 +1,45 @@
 import pyquery
 from random import randint
 
-# Functions that are used across different locustfiles
-def authenticity_token(dom, id=None):
+# Utility functions that are used across different locustfiles
+
+def get_request(context, path, expected_redirect=None):
+    """
+    This is a generic get request.
+    """
+    with context.client.get(path, 
+        headers=desktop_agent_headers(),
+        catch_response=True) as resp:
+            if expected_redirect:
+                verify_resp_url(expected_redirect, resp)
+            dom = resp_to_dom(resp)
+
+            return dom
+
+def post_request(context, path, data, expected_redirect=None):
+    with context.client.post(
```

Comment:
maybe we could add a `method` param and then do like like:

```python
method = 'post'
getattr(context.client, method)(path, headers, data) # etc etc
```

Reply:
This was super helpful, thank you. `getattr` was a thing I needed.

Reply:
:high-five: !